### ISSUE TITLE
What happend? Segmentation fault: 11

### ISSUE THREAD

**Issue by qiqcheng** [TuSimple/simpledet#277]
![TIM截图20191216211742](https://user-images.githubusercontent.com/20226330/70909983-ad467a00-2049-11ea-8345-d3dc85945dfb.jpg)

**Comment by qiqcheng** [comment]
training with rpn_r50v2c4_1x.py ..

**Comment by RogerChern** [comment]
Is this reproducible?
Could you please provide some informations about your platform?

**Comment by shouwangzhe134** [comment]
build mxnet==1.6.0 from source

**Comment by RogerChern** [comment]
Well, there is a known bug in the Proposal operator that will trigger
segment fault when RPN gives some invalid output like NANs. So please try
to lower the learning rate or the loss scale of the RPN head.

**Comment by XieXing9212** [comment]
> build mxnet==1.6.0 from source

how to build mxnet==1.6.0 from source? I follow the official install guidline, but I meet  segmentation fault when I test simpledet.

**Comment by dongzhenguo2016** [comment]
I recently encountered the same error while training cascade_r101v1_fpn_1x, how can I solve it? Feel so strange.
My platform is ubuntu 16.04
maxnet-cu100  1.6.0
